AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/PackageManifestBuilder.swift

```diff
@@ -107,12 +111,6 @@ struct PackageManifestBuilder {
             // Add the generated content that defines the list of services to include
             buildServiceTargets(),
             "",
-            // Add the dependencies for the internal clients
-            buildInternalAWSSTSDependencies(),
-            buildInternalAWSSSODependencies(),
-            buildInternalAWSSSOOIDCDependencies(),
-            buildInternalAWSCognitoIdentityDependencies(),
-            "",
```

Comment:
Here & below, special code for internal services is removed since they are now handled like all other services and filtered as needed into public/internal at runtime.